runtime/doc/msgpack_rpc.txt

```diff
@@ -221,7 +221,10 @@ Special types (msgpack EXT) ~
 
 An API method expecting one of these types may be passed an integer instead,
 although they are not interchangeable. For example, a Buffer may be passed as
-an integer, but not a Window or Tabpage.
+an integer, but not a Window or Tabpage. The data in the EXT object is the
```

Comment:
The old sentence is confusing. Should it be:

> For example, a Buffer may be passed as an integer, but not *as* a Window or Tabpage.

Or, is it saying Window/Tabpage may not be passed as integers, in contrast to Buffers?